phpunit.xml.dist

```diff
@@ -41,6 +41,7 @@
 		</testsuite>
 		<testsuite name="shortcodes">
 			<directory prefix="test" suffix=".php">tests/php/modules/shortcodes</directory>
+			<exclude>tests/php/modules/shortcodes/test-class.instagram.php</exclude>
```

Comment:
Some of these tests seem to be passing. Should we mark only the failing tests as skipped?

Also it would be great if we could do it only when `WP_BRANCH` is `master`.

Finally, I'd suggest making sure we have a @TODO comment that will remind us to fix them and enable them if we happen to forget.

Reply:
> I'd suggest making sure we have a @todo comment that will remind us to fix them and enable them if we happen to forget.

Prolly safest to open a new issue so that Stale bot comes back ~yelling at~ reminding us if we forget. :-)

Reply:
A new issue in the 7.6 branch works for me 👍

Reply:
https://github.com/Automattic/jetpack/issues/12918